internal/app/engine/config.go

```diff
@@ -15,6 +15,7 @@ var defaultTeams = []string{
 	"AMC",
 	"A-Team",
 	"CMμs",
+	"Delft Mercurians",
```

Comment:
Spelling is important here. Now you have a space in the name, but in the status-board, you don't. This will not work.
Either remove the space here, or create another PR changing the name in the status-board (spaces are replaced with `-` there).